assets/js/src/public.jsx

```diff
@@ -2,50 +2,91 @@ import MailPoet from 'mailpoet';
 import jQuery from 'jquery';
 import 'parsleyjs';
 
-jQuery(function ($) { // eslint-disable-line func-names
+function setCookie(name, value, days) {
+  let expires = '';
+  if (days) {
+    const date = new Date();
+    date.setTime(date.getTime() + (days * 24 * 60 * 60 * 1000));
+    expires = `; expires=${date.toUTCString()}`;
+  }
+  document.cookie = `${name}=${value}${expires}; path=/`;
+}
+function getCookie(name) {
```

Comment:
I know this is from SO, but maybe at least some reasonable variable names inside this function so it's readable for devs? It'll be minified when compiled.